codegen/build.gradle

```diff
@@ -1,9 +1,17 @@
 
 description 'web3j project code generators'
-
+repositories {
+    maven { url 'http://dl.bintray.com/web3j/maven' }
```

Comment:
Why do we need the bintray repos here?

Reply:
This will be changed once there is a release for web3j-unit which should be before this PR is merged. Currently, if I follow the instructions at [web3j-unit](https://github.com/web3j/web3j-unit) I'm not able to get the dependencies.